triggers/runner/src/main/resources/logback.xml

```diff
@@ -1,9 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
-    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder>
-            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg %replace(, context: %marker){', context: $', ''} %n</pattern>
-        </encoder>
+    <appender name="STDOUT" class="net.logstash.logback.appender.LoggingEventAsyncDisruptorAppender">
+        <appender class="ch.qos.logback.core.ConsoleAppender">
+            <if condition='isDefined("LOG_FORMAT_JSON")'>
+                <then>
+                    <encoder class="net.logstash.logback.encoder.LoggingEventCompositeJsonEncoder">
+                        <pattern>
+                            {
+                            "level": "%level",
+                            "logger": "%logger{36}",
+                            "thread": "%thread",
+                            "summary": "%msg"
```

Comment:
IIRC, we use `summary` here to avoid confusing the JSON encoder (as `message` is already known to it)